lua/wire/client/e2descriptions.lua

```diff
@@ -375,6 +375,18 @@ E2Helper.Descriptions["getCollisionGroup(e:)"] = "Returns the collision group of
 E2Helper.Descriptions["setCollisionGroup(e:n)"] = "Sets the collision group of the entity. Does not work on players. Use one of the _COLLISION_GROUP constants"
 E2Helper.Descriptions["noCollideAll"] = "Nocollides an entity to all entities/players, just like the tool's right-click"
 
+-- Movedata/Usercmd
+E2Helper.Descriptions["getMouseDeltaX()"] = "Returns the mouse delta of this command on the x-axis"
```

Comment:
Need the meta type in the arguments for the signature to be correct:

``getMouseDeltaX(xuc:)``

Reply:
damn it, well at least i know now.

Reply:
ok done.